{
  "id": "EUVD-2019-10980",
  "enisaUuid": "2a7f0891-e083-3a8d-81e8-e7ef1e997c46",
  "description": "In the Lustre file system before 2.12.3, the mdt module has an out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. mdt_file_secctx_unpack does not validate the value of name_size derived from req_capsule_get_size.",
  "datePublished": "Jan 27, 2020, 4:19:42 AM",
  "dateUpdated": "Aug 5, 2024, 2:39:09 AM",
  "baseScore": 7.5,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
  "references": [
    "http://wiki.lustre.org/Lustre_2.12.3_Changelog",
    "http://lustre.org/",
    "https://jira.whamcloud.com/browse/LU-12604",
    "https://review.whamcloud.com/#/c/35868/"
  ],
  "aliases": [
    "CVE-2019-20432",
    "GSD-2019-20432"
  ],
  "assigner": "mitre",
  "epss": 0.61,
  "enisaIdProduct": [
    {
      "id": "d21f1f71-d6ff-3249-be84-4a658859d1a0",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "bbc94ef5-79c6-3789-b480-60e69a361071",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2019-20432"
}